src/constants.ts

```diff
@@ -7,3 +9,24 @@ export const OPENSEA_BASE_NFTS_URL = `${OPENSEA_BASE_URL}/assets`;
 export const MAGICEDEN_BASE_URL = 'https://magiceden.io';
 export const MAGICEDEN_BASE_COLLECTIONS_URL = `${MAGICEDEN_BASE_URL}/marketplace`;
 export const MAGICEDEN_BASE_NFTS_URL = `${MAGICEDEN_BASE_URL}/item-details`;
+
+const evmLabels = {
+    collectionId: "Contract Address",
+    tokenId: "Token ID",
+    tokenStandard: "ERC721"
+};
+
+/**
+ * Labels for collection, token, and token standards per-chain.
+ * eg: on EVM chains, we have a "Contract Address", but on Solana that's
+ * referred to as an "On-chain Collection"
+ */
+export const chainIdentifierLabels: Record<Chain, { collectionId: string; tokenId: string; tokenStandard: string; }> = {
```

Comment:
i thought about it, but when i thought about whether to have the type in this file, or in `types.ts` and import, my brain turned off. Any opinion on whether to have the type locally or in the `types` file?

Reply:
I would be happy with putting the type in `types/general.ts` for now or creating a dedicated `types/chain.ts` -- no preference on which to do, probs the former since easier. Another advantage of abstracting and exporting this as a type is that with my current refactor, I could totally see this type being a component prop.